keyboards/afternoonlabs/oceanbreeze/rev1/info.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Sorry, the thumb keys still seem to be slightly out of order:
![image](https://user-images.githubusercontent.com/4781841/112431889-32aff280-8d94-11eb-9806-69488849b7ec.png)